MIGRATION.md

```diff
@@ -29,3 +30,260 @@ import '../../node_modules/@metamask/design-tokens/dist/styles.css';
+--brand-colors-violet
+--brand-colors-white-white010
+--color-overlay-inverse
+--color-primary-shadow
+--color-primary-disabled
+--color-secondary
+--color-error-shadow
+--color-error-disabled
+--color-warning-alternative
+--color-warning-disabled
+--color-success-alternative
+--color-success-disabled
+--color-info-alternative
+--color-info-disabled
+--color-network
+```
+
+#### JS
+
+```
+brandColor.grey030
+brandColor.grey040
+brandColor.grey750
+brandColor.blue000
+brandColor.green000
+brandColor.red000
+brandColor.yellow000
+brandColor.orange000
+brandColor.violet
+brandColor.white010
+lightTheme.colors.overlay.inverse
+lightTheme.colors.primary.shadow
+lightTheme.colors.primary.disabled
+lightTheme.colors.secondary
+lightTheme.colors.error.shadow
+lightTheme.colors.error.disabled
+lightTheme.colors.warning.alternative
+lightTheme.colors.warning.disabled
+lightTheme.colors.success.alternative
+lightTheme.colors.success.disabled
+lightTheme.colors.info.alternative
+lightTheme.colors.info.disabled
+lightTheme.colors.network
+
+// TO DO add network colors
+// TO DO add dark theme colors
+```
+
+### Changed
+
+#### CSS
+
+```
+--brand-colors-white-white000` to `--brand-colors-white
+--brand-colors-black-black000` to `--brand-colors-black
+```
+
+#### JS
+
+```
+brandColor.white000` to `brandColor.white
+brandColor.black000` to `brandColor.black
+```
+
+### Brand Colors
+
+- Grey updates
+- grey000 added
+- grey025 added
+- grey050 added
+- grey030 removed
+- grey040 removed
+- grey750 removed
+- grey1000 added
+
+- Blue updates
+- blue000 removed
+- blue025 added
+- blue050 added
+- blue400-15 added
+- blue500-10 added
+
+- Green updates
+- green000 removed
+- green025 added
+- green050 added
+- green400-15 added
+- green500-10 added
+
+- Red updates
+- red000 removed
+- red025 added
+- red050 added
+- red400-15 added
+- red500-10 added
+
+- Yellow updates
+- yellow000 removed
+- yellow025 added
+- yellow050 added
+- yellow100-15 added
+- yellow400-15 added
+- yellow500-10 added
+- yellow700 added
+- yellow800 added
+- yellow900 added
+
+- Orange updates
+- orange000 removed
+- orange025 added
+- orange050 added
+
+- Violet removed
+
+- Purple added
+
+- Lime added
+
+- White and black updates
+- white000 renamed to white
+- white010 removed
+- black000 renamed to black
+
+### Brand Colors Brand Evolution
+
+- Blue updates
+- blue300-15 added
+- blue400-15 removed
+
+- Green updates
+- green300-15 added
+- green400-15 removed
+
+- Red updates
+- red300-15 added
+- red400-15 removed
+
+### Themed Colors
+
+- overlay-inverse removed
+- primary-shadow removed
+- primary-disabled removed
+- secondary colors removed
+- error-shadow removed
+- error-disabled removed
+- warning-alternative removed
+- warning-disabled removed
+- success-alternative removed
+- success-disabled removed
+- info-alternative removed
+- info-disabled removed
+- network colors removed
+- shadow-primary added
+- shadow-error added
+- button colors removed
```

Comment:
These can be removed once they have be cross referenced with the JS and CSS above

Reply:
@georgewrmarshall you'll need to be a little more specific. I'm not completely understand how you want this written out

Reply:
I've listed the specific JS and CSS tokens that engineers should check for. For example, what does "button colors" mean? What do "network colors" refer to? Without these specifics, it's difficult for engineers to understand what to look for when identifying removed design token code. This list should also be used to check for any removed CSS variables.